appbuilder/services/livesync/livesync-service.ts

```diff
@@ -0,0 +1,102 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import { exportedPromise } from "../../../decorators";
+
+export class ProtonLiveSyncService implements IProtonLiveSyncService {
+	private excludedProjectDirsAndFiles = ["app_resources", "plugins", ".*.tmp", ".ab"];
+
+	private get $liveSyncServiceBase(): ILiveSyncServiceBase {
```

Comment:
As this class is  called `ProtonLiveSyncService` couldn't it `extend` `LiveSyncServiceBase` instead of resolving it as a `get` property